lib/commands/deploy.ts

```diff
@@ -125,6 +127,11 @@ ${dockerignoreHelp}
 			description:
 				"don't upload build logs to the dashboard with image (if building)",
 		}),
+		'release-version': flags.string({
+			description: stripIndent`
+				Set release version to uniquely identify this release within your app.
```

Comment:
```suggestion
				Set a release version to uniquely identify this release within your app.
```

Also, kindly remind me: does the version have to be valid semver? Or any arbitrary string will do? If the former, we could add input validation in the CLI.